src/mongo/delegates/Tables.class.php

```diff
@@ -554,8 +553,11 @@ public function generateTableRows($tableType, $resource = null, $context = null,
                     $from,
                     array($tableType)
                 );
-
-                $this->getApplyOperation()->createJob(array($subject), $queueName, $jobOptions);
+                $subjects[] = $subject;
```

Comment:
Just a thought (and not a blocker for this), but is it worth putting this logic in a class higher up the food chain / separate class/util rather than repeated in each composite?

Reply:
Probably, but all of this is kind of hairy, anyway.  I think in general there is a lot of consolidation that could be done between the composites to simplify them.